### ISSUE TITLE
Service Fabric - Various Process 

### ISSUE THREAD

**Issue by nareshkhatri81** [Azure/service-fabric-issues#601]
Hi Team,
i am seeing lots of service which are running in background in SF Nodes.
is there documentation of what is purpose of those service ?
i am looking for naming service and reverse proxy service. where i can see them and how can kill them for simulating failover ? i tried with SF explore by disabling node but i was able to get to service via service fabric reverse proxy ?

Below are service which i could see running in node in background. Fabric has multiple instances.
Fabric
Fabric
FabricApplicationGateway
FabricFAS
FabricHOST
FabricTypeHost
FabricUOS
FileStoreService

Thanks,
Naresh Khatri

**Comment by vturecek** [comment]
These are various host processes for system services. I don't recommend messing with these. More info on architecture here: https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-architecture

There are fault injection commands you can use to simulate failures. Start here: https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-controlled-chaos

**Comment by nareshkhatri81** [comment]
Yes but knowing about these processed will help to investigate and trouble any error.
Reverse Proxy service doesnt show up in Service Fabric Explorer. so how we can know if its healthy and running fine on each node ?
